powertools-logging/powertools-logging-log4j/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>powertools-parent</artifactId>
+        <groupId>software.amazon.lambda</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>powertools-logging-log4j</artifactId>
+    <packaging>jar</packaging>
+    <name>Powertools for AWS Lambda (Java) library Logging with Log4j2</name>
+    <description>Set of utility for better logging with log4j</description>
+
+    <url>https://aws.amazon.com/lambda/</url>
+    <issueManagement>
```

Comment:
Semi-related: Do we really need this duplication (`issueManagement`) in _every_ child?

Reply:
don't think so, but I think it is in all our poms...